### DISCUSSION TITLE
禁止跳转动态详情相关

### DISCUSSION THREAD

**Discussion by RtYkk** [the1812/Bilibili-Evolved#4451] [功能建议 / Ideas]
我记得原来 a 转发 b 的动态我想进 b 的动态详情是不是有个 ">查看详情" 的小灰色文字的，批站好像又把这个改了，现在只能点击动态本体才能查看详情了……很烦。这样禁止跳转动态详情和想看看 b 动态不可兼得了，有点麻烦。

**Comment by Geeyun-JY3** [comment]
你说的这个被转发动态下的`详情 >`灰色文字按钮就是`禁止跳转动态详情`组件的功能：https://github.com/the1812/Bilibili-Evolved/blob/602a80dd007340cbb61328442dd5cf2098f49cb7/registry/lib/components/feeds/disable-details/index.ts#L80
这个组件这部分功能近期失效了，近期也有人开过一个 PR：#4433

**Reply by RtYkk** [reply]
草，才知道。但我看这个 pr 关了，说应该恢复正常了？但我这还是没有。

**Reply by Geeyun-JY3** [reply]
我这边实测还是没恢复的，不知道 PR 发起者为什么关了。
